package-lock.json

```diff
@@ -1,175 +1,1770 @@
 {
```

Comment:
We should ignore this since this files is managed by npm, not by hand

Reply:
Pretty sure I did touch it via npm only, no prettier or eslint included. Will regenerate it to ensure this :)